Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector_test.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import demistomock as demisto
+import io
+from AbnormalSecurityEventCollector import get_threats
+from CommonServerPython import *
+from Packs.AbnormalSecurity.Integrations.AbnormalSecurityEventCollector.AbnormalSecurityEventCollector import Client
+
+
+class MockResponse:
+    def __init__(self, data, status_code):
+        self.data = data
+        self.text = str(data)
+        self.status_code = status_code
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+class Client(BaseClient):
+    def get_threats(self, params):
+        return util_load_json('test_data/test_get_list_threats.json')
+
+    def get_threat(self, threat):
+        return util_load_json('test_data/test_get_threat.json').get(threat)
+
+"""
+    Command Unit Tests
+"""
+
+
+def test_get_a_list_of_threats_command():
+    """
```

Comment:
Please add more info to this docstring as you have some more details bout your test conditions, like the fact your only searching for messages older than the 'after' key.